.github/cloudbuild.samples.yaml

```diff
@@ -0,0 +1,7 @@
+steps:
+- name: node:14
+  entrypoint: npm
+  args: ['install', '--unsafe-perm']
```

Comment:
@donmccasland I needed to use `--unsafe-perm` here because by default Cloud Build appears to run containers as a root user.  Do you know if there's a way to avoid this?

Reply:
Justin noted that there's no Dockerfile here, so no way to manipulate the USER variable.  One potential solution might be:
steps:
- name: node:14
  entrypoint: /bin/bash
  args: 
      - -c
      - |
          USER=joe npm install